README.md

```diff
@@ -1 +1,135 @@
-# module-1-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [IWES conference 2023] <a name="about-project"></a>
+
+- This project is called IWES-Conference-2023. It is based on a conference that is to happen towards the end of this year in Abuja, Nigeria for women all over the world. This project contains all information about the coming conference and the previous ones as well.
+
+**[IWES conference 2023]** is a website that is used to tell women about this program and at the same time invite them for the program.
+
+## 🛠 Built With <a name="HTML and CSS and JS"></a>
+
+### Tech Stack <a name="Front end (Javascript)"></a>
+
+- This is a mobile and desk-top browser responsive website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.w3schools.com/js/">JS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[It is a two paged website ]**
+- **[It has a navbar for navigation to the pages]**
+- **[There is a join button that when focused on tells you the amount of slots remaining for the program]**
+- **[This website is mobile and desktop responsive and this conclusion came from testing it on most devices]**
+- **[There is an active class that shows what page you are currently on]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+🚀 Live Demo
+
+- (Live Demo to be inputted after it PR has been approved in case of any changes)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Great job on creating the README file for your project! :+1: However, it's important to ensure that you deploy your project and add the link to the live version in your README file, even if you haven't finished your project yet. This is important because it will allow others to easily access and view your project, as well as provide you with a way to test your project on a live server. Additionally, it will be automatically updated with every change that you push to your GitHub repository. Therefore, it's important to deploy your project and add the link to the live version in your README file to provide easy access to your project.

Reply:
Thank you, I have deployed and added the link to my README file.